pkg/storer/internal/reserve/reserve.go

```diff
@@ -345,7 +345,7 @@ func removeChunk(ctx context.Context, store internal.Storage, item *batchRadiusI
 		return err
 	}
 
-	err := indexStore.Delete(&chunkBinItem{Bin: item.Bin, BinID: item.BinID})
+	err := indexStore.Delete(&chunkBinItem{Bin: item.Bin, BinID: item.BinID, Address: item.Address})
```

Comment:
So I saw this error on the test cluster. The problem is we are not doing a Get in the TxStore.Delete. This will need to be fixed. Basically, the current assumption in the code is that for Delete we only need to populate the Key interface for the item. So the TxStore should do a get first and fill all the fields. Otherwise the rollback operations will populate incorrect items. For eg. in this case, it will give error for missing address, but batch ID will be nil in case of rollback.Put.

Reply:
Done.